dissect/target/plugins/os/windows/log/evtx.py

```diff
@@ -140,13 +146,18 @@ def _build_record(self, evtx_record) -> Record:
                 if key == "EventID":
                     value = int(value)
 
+                if key in (keys := set(record_values.keys())):
```

Comment:
```suggestion
                if key in record_values:
```
Don't think you need the set or `.keys()`, can just do it in the dictionary directly.

Need to find an appropriate type hint though.